compiler/rustc_mir/src/transform/const_prop.rs

```diff
@@ -476,6 +478,19 @@ impl<'mir, 'tcx> ConstPropagator<'mir, 'tcx> {
             Ok(op) => Some(op),
             Err(error) => {
                 let tcx = self.ecx.tcx.at(c.span);
+                if error.kind().is_spurious() {
+                    // Spurious errors can't be ignored since otherwise the amount of available
```

Comment:
I don't like the name "spurious"...

Reply:
"volatile" sounds like it is related to `read_volatile`/`write_volatile`... maybe let's discuss a bit instead of you just posting new commits all the time. :)
Actually I hope we can just entirely [avoid even having this method](https://github.com/rust-lang/rust/pull/86255#discussion_r661471723).